integrations/micrometer/micrometer/src/main/java/module-info.java

```diff
@@ -32,17 +32,17 @@
 
     requires java.logging;
 
-    requires static jakarta.annotation;
+    requires jakarta.annotation;
 
     requires io.helidon.config;
```

Comment:
This is part of public API, should be a transitive dep.

Reply:
There should also be a `requires transitive io.helidon.common`, as the API uses `Builder` from common module.

Reply:
ok